Transport/Simple/Transport.H

```diff
@@ -11,427 +11,778 @@
+        Omega_M += Xloc[i] * Xloc[j] * Omega_ij * T3;
+
+        amrex::Real MW_ij =
+          2.0 / (trans_parm->trans_iwt[i] + trans_parm->trans_iwt[j]);
+        MW_m += Xloc[i] * Xloc[j] * Epsilon_ij * T2 * std::sqrt(MW_ij);
+
+        DP_m_4 += Xloc[i] * Xloc[j] * trans_parm->trans_dip[i] *
+                  trans_parm->trans_dip[i] * trans_parm->trans_dip[j] *
+                  trans_parm->trans_dip[j] / (T3 * Epsilon_ij);
+
+        KappaM += Xloc[i] * Xloc[j] *
+                  std::sqrt(trans_parm->Kappai[i] * trans_parm->Kappai[j]);
+      }
+    }
+
+    MW_m *= MW_m;
+    amrex::Real sigma_M = std::cbrt(sigma_M_3);
+    amrex::Real InvSigma1 = 1.0 / sigma_M;
+    amrex::Real InvSigma3 = 1.0 / sigma_M_3;
+    Epsilon_M *= InvSigma3;
+    amrex::Real Tstar = Temp / Epsilon_M;
+    amrex::Real Tcm = 1.2593 * Epsilon_M;
+    amrex::Real Vcm = 1.8887 * sigma_M_3;
+    Omega_M *= InvSigma3;
+    MW_m = MW_m * InvSigma3 * InvSigma1 / (Epsilon_M * Epsilon_M);
+    DP_m_4 = DP_m_4 * sigma_M_3 * Epsilon_M;
+    amrex::Real DP_red_4 = 297.2069113e6 * DP_m_4 / (Vcm * Vcm * Tcm * Tcm);
+    amrex::Real y = Vcm * rholoc / (6.0 * wbar);
+    amrex::Real A[10];
+    for (int i = 0; i < 10; ++i) {
+      A[i] = trans_parm->Afac[4 * i] + trans_parm->Afac[4 * i + 1] * Omega_M +
+             trans_parm->Afac[4 * i + 2] * DP_red_4 +
+             trans_parm->Afac[4 * i + 3] * KappaM;
+    }
+    amrex::Real G1 = (1.0 - 0.5 * y) / ((1.0 - y) * (1.0 - y) * (1.0 - y));
+    amrex::Real G2 = (A[0] * (1.0 - std::exp(-A[3] * y)) / y +
+                      A[1] * G1 * std::exp(A[4] * y) + A[2] * G1) /
+                     (A[0] * A[3] + A[1] + A[2]);
+    amrex::Real eta_P =
+      (36.344e-6 * std::sqrt(MW_m * Tcm) / std::pow(Vcm, 2.0 / 3.0)) * A[6] *
+      y * y * G2 * exp(A[7] + A[8] / Tstar + A[9] / (Tstar * Tstar));
+
+    // Set nonideal viscosity
+    if (wtr_get_mu) {
+      mu = mu * (1.0 / G2 + A[5] * y) + eta_P;
+    }
 
-  amrex::Real B[7];
-  for (int i = 0; i<7; ++i) {
-    B[i] = trans_parm->Bfac[i*4] + trans_parm->Bfac[i*4+1]*Omega_M + trans_parm->Bfac[i*4+2]*DP_red_4 + trans_parm->Bfac[i*4+3] * KappaM;
+    amrex::Real B[7];
+    for (int i = 0; i < 7; ++i) {
+      B[i] = trans_parm->Bfac[i * 4] + trans_parm->Bfac[i * 4 + 1] * Omega_M +
+             trans_parm->Bfac[i * 4 + 2] * DP_red_4 +
+             trans_parm->Bfac[i * 4 + 3] * KappaM;
+    }
+    amrex::Real H2 = (B[0] * (1.0 - std::exp(-B[3] * y)) / y +
+                      B[1] * G1 * std::exp(B[4] * y) + B[2] * G1) /
+                     (B[0] * B[3] + B[1] + B[2]);
+
+    amrex::Real lambda_p = 3.039e-4 * std::sqrt(Tcm / MW_m) /
+                           std::pow(Vcm, 2.0 / 3.0) * B[6] * y * y * H2 *
+                           std::sqrt(Tstar);
+
+    lambda_p *= 4.184e+7; // erg/(cm s K)
+    amrex::Real beta = 1.0 / H2 + B[5] * y;
+
+    // Set nonideal conductivity
+    if (wtr_get_lam) {
+      lam = lam * beta + lambda_p;
+      // lam = H2;
```

Comment:
extraneous commented code left over from testing previous PR, can be deleted

Reply:
done